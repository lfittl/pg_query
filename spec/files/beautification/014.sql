SELECT x FROM a join x on a.y = x.y join b ON a.c = b."id" AND a.d = b.d AND ( ( a.e >= 123 AND a.e <= 234) OR ( a.f = 'x' AND a.g <= 500))
