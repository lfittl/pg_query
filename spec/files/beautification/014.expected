SELECT
  "x"
FROM
  "a"
  JOIN "x" ON ("a"."y" = "x"."y")
  JOIN "b" ON "a"."c" = "b"."id" AND
              "a"."d" = "b"."d" AND
              (
                (
                  "a"."e" >= 123 AND
                  "a"."e" <= 234
                ) OR
                (
                  "a"."f" = 'x' AND
                  "a"."g" <= 500
                )
              )
